boot/freeldr/freeldr/arch/realmode/i386.S

```diff
@@ -25,54 +25,55 @@ RealModeEntryPoint:
     mov ss, ax
 
     /* Setup the stack */
-    mov sp, word ptr ds:[stack16]
+    mov sp, STACK16ADDR
 
     /* Enable A20 address line */
     call EnableA20
 
+    /* Get current base address */
```

Comment:
or `/* Update BSS_CurrentBaseAddress */`
(we would automatically know where this get stored etc.)
But it seems that the GetCurrentBaseAddress function does more than that...

Reply:
done i updated the comment